tests/system/providers/google/marketing_platform/example_analytics.py

```diff
@@ -29,7 +29,10 @@
     GoogleAnalyticsModifyFileHeadersDataImportOperator,
     GoogleAnalyticsRetrieveAdsLinksListOperator,
 )
+from airflow.utils.trigger_rule import TriggerRule
 
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = "example_google_analytics"
 ACCOUNT_ID = os.environ.get("GA_ACCOUNT_ID", "123456789")
 
 BUCKET = os.environ.get("GMP_ANALYTICS_BUCKET", "test-airflow-analytics-bucket")
```

Comment:
The migration of the system tests also involves moving the part created in the pytest to the DAG itself. For example in this file:
https://github.com/apache/airflow/pull/24237/files#diff-69116730de9cf0f546bfcebd1c7023fb8342739adb56822af5c73215c0db8d41

You can see that there is bucket created and removed before & after the test. For the migration we also need to do it inside DAG (add create bucket / delete bucket operators). See this PR as example: https://github.com/apache/airflow/pull/23247